multi-agent/lib/mas/blueprints/resolver.py

```
@@ -63,29 +72,31 @@ def _stash_inline(self, cat: ResourceCategory, res: BlueprintResource, bucket: d
             )
         )
         # still inspect it for nested rids
-        self._scan_nested(concrete, bucket, visited)
+        self._scan_nested(concrete, bucket, visited, identity)
 
-    def _walk_live(self, rid: str, name: str | None, bucket: dict, visited: set):
-        """Fetch a live resource and recurse through its config."""
+    def _walk_live(
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_walk_live`

Add return type annotation: `None`

(ANN202)